### ISSUE TITLE
DDLDatabase fails with COMMENT on columns

### ISSUE THREAD

**Issue by anuraaga** [jOOQ/jOOQ#7018]
Parser does not seem to recognize COMMENT inside CREATE TABLE

JOOQ 3.10.3

```
org.jooq.impl.ParserException: C42000_NO_SUBCLASS: Expected tokens: [AUTOINCREMENT, AUTO_INCREMENT, CHECK, DEFAULT, GENERATED, NOT NULL, NULL, PRIMARY KEY, UNIQUE]: 
  max_elevation DOUBLE NOT NULL,
  skylines BLOB [*]COMMENT 'skyline azimuth and elevations, protobuf list of Angle',
  tripzones BL
        at org.jooq.impl.ParserImpl$ParserContext.unexpectedToken(ParserImpl.java:5791)
        at org.jooq.impl.ParserImpl.parseQuery(ParserImpl.java:547)
        at org.jooq.impl.ParserImpl.parse(ParserImpl.java:345)
        at org.jooq.impl.ParserImpl.parse(ParserImpl.java:337)
        at org.jooq.util.ddl.DDLDatabase.create0(DDLDatabase.java:124)
        at org.jooq.util.AbstractDatabase.create(AbstractDatabase.java:237)
        at org.jooq.util.AbstractDatabase.create(AbstractDatabase.java:226)
        at org.jooq.util.AbstractDatabase.getDialect(AbstractDatabase.java:208)
        at org.jooq.util.AbstractGenerator.logDatabaseParameters(AbstractGenerator.java:131)
        at org.jooq.util.JavaGenerator.generate(JavaGenerator.java:271)
        at org.jooq.util.GenerationTool.run(GenerationTool.java:642)
        at org.jooq.util.GenerationTool.generate(GenerationTool.java:203)
        at org.jooq.util.GenerationTool.main(GenerationTool.java:175)
```

**Comment by lukaseder** [comment]
Thanks for reporting.

While jOOQ 3.11 will support the SQL standard `COMMENT ON { TABLE | COLUMN }` syntax (#6324), it is not yet possible to apply comments to columns when running DDL on MySQL, which doesn't support the standard syntax (see #6995).

It would definitely make sense to parse the syntax, though.

**Comment by lukaseder** [comment]
This is now fixed for jOOQ 3.11 and will be merged to 3.10.4 next week (#7029).

**Comment by lukaseder** [comment]
This fix depends on #7027, which is also available only in 3.11
